models.py

```diff
@@ -41,9 +43,30 @@ def serialize(self):
         'celestial_body_type': self.celestial_body_type,
         'gravity': self.gravity,
         'planet_day': self.planet_day,
-        'planet_year': self.planet_year
+        'planet_year': self.planet_year,
+        'travel': self.travel_time()
       }
 
+    def travel_time(self):
+      miles_per_au = 92955807 # Number of miles in one astronomical unit
+      ship_speed = 24816 #Ship speed equal to maximum speed for manned spaceflight
```

Comment:
We talked about what might be best for a speed figure, considering that figuring out space travel is well... they don't call it rocket science for nothing.